lastminutewf2020.yml

```diff
@@ -36,6 +37,13 @@
         mode: '0644'
 - hosts: all
   tasks:
+    - name: Update all packages to their latest version
+      apt:
+        name: "*"
```

Comment:
I think this will get every package there is, and set the status to be latest, probably we want this also:
```
- only_upgrade
        Only upgrade a package if it is already installed.
        [Default: no]
        type: bool
        version_added: 2.1
```

Reply:
added